components/connect-button.tsx

```diff
@@ -22,6 +25,7 @@ export const ConnectWalletButton = () => {
         const connected = mounted && account && chain;
 
         const sendTransactionHandler = async () => {
+          displaySwapSuccess(SALE_SUCCESSFUL);
```

Comment:
I guess we don't need a variable for this, you can just pass true directly on params